Core/PixelEvent.swift

```diff
@@ -505,6 +505,9 @@ extension Pixel {
         case dbCrashDetected
         case crashOnCrashHandlersSetUp
 
+        case crashReportCRCIDMissing // crashreporting_crcid-missing
+        case crashReportingSubmissionFailed // crashreporting_submission-failed
```

Comment:
We can omit the comments here for consistency, since the value below will contain the raw name.

Reply:
Thanks for catching this, it some leftover note taking from when I was working it all out :)  I’ve removed from both iOS and macOS clients and will push shortly.